scss/components/_card.scss

```diff
@@ -67,7 +67,7 @@ $card-margin: $global-margin !default;
   box-shadow: $shadow;
 
   overflow: hidden;
-  color: $card-font-color;
+  color: $color;
```

Comment:
See -> [Line 50](https://github.com/IngoVals/foundation-sites/blob/f9e9d94a2f065ec84812ffc56731c0fcfa33e840/scss/components/_card.scss#L50). Within the mixin, this is already set with settings variable

```scss
$color: $card-font-color
```

which means if somebody writes this below .... then that color will be equal to `card-font-color`

```scss
.mycard {
   @include card-container;
}
```

But also, if you dont want that, then it can be customized within a mixin like 

```scss
.mycard {
   @include card-container($color: #ccc);
}
```

This is just how it goes to reuse mixin in sass!
See => https://www.sitepoint.com/sass-basics-the-mixin-directive/